pkg/client/transport/round_trippers.go

```diff
@@ -133,22 +135,48 @@ func (rt *basicAuthRoundTripper) CancelRequest(req *http.Request) {
 
 func (rt *basicAuthRoundTripper) WrappedRoundTripper() http.RoundTripper { return rt.rt }
 
+const (
+	// ImpersonateUserHeader is used to impersonate a particular user during an API server request
```

Comment:
note that these correspond to the headers used in `pkg/apis/authentication`... I assume you're not importing those because you don't want the package dependency